solidity/contracts/isms/MultisigIsm.sol

```diff
@@ -25,6 +25,10 @@ contract MultisigIsm is IMultisigIsm, Ownable {
     using MultisigIsmMetadata for bytes;
     using MerkleLib for MerkleLib.Tree;
 
+    // ============ Constants ============
```

Comment:
Optional, but might be nice to have test coverage for spec compliance, though i can see how it feels so trivial

Reply:
Yeah I just wanted to get something out last night so that you'd be able to review. I can add